front/ui/ui-core/src/components/inputs/datePicker/__tests__/useCalendar.spec.ts

```diff
@@ -146,27 +146,20 @@ describe('useCalendar', () => {
     });
   });
 
-  describe('isReferenceDate', () => {
-    const today = new Date();
-    today.setHours(0, 0, 0, 0);
-    const { result } = renderHook(() =>
-      useCalendar({
-        displayedMonthStartDate,
-        selectedSlot,
-        onDayClick: () => {},
-      })
-    );
-
+  describe('isToday', () => {
     it('should return true for today', () => {
-      expect(result.current.isReferenceDate(today)).toBe(true);
+      const today = new Date();
+      expect(defaultResult.current.isToday(today)).toBe(true);
     });
 
-    it('should return true for the selectableSlot start date', () => {
-      expect(defaultResult.current.isReferenceDate(selectableSlot.start)).toBe(true);
+    it('should return false for a date in the past', () => {
+      const pastDate = new Date(2020, january, 1);
+      expect(defaultResult.current.isToday(pastDate)).toBe(false);
     });
 
-    it('should return false for a day that is not today', () => {
-      expect(result.current.isReferenceDate(new Date(2023, july, 15))).toBe(false);
+    it('should return false for a date in the future', () => {
+      const futureDate = new Date(2025, january, 1);
+      expect(defaultResult.current.isToday(futureDate)).toBe(false);
```

Comment:
You only need to test for a date which is not today, no need to check for a date in the past and a date in the future